README.md

```diff
@@ -96,9 +112,12 @@ dir.readFiles(__dirname, {
     function(err, files){
         if (err) throw err;
         console.log('finished reading files:',files);
-    });
+    }
+);
+```
 
-// exclude an array of subdirectory names
+Exclude an array of subdirectory names
+```
 dir.readFiles(__dirname, {
     exclude: ['node_modules', 'test']

```

Comment:
This is the typo in the actual README: the option to exclude directories is `excludeDir` instead of `exclude`.